### ISSUE TITLE
make pixbufloader error

### ISSUE THREAD

**Issue by lehitoskin** [FLIF-hub/FLIF#419]
I tried compiling the new pixbufloader and got this error:
```
cc -shared -I/usr/include/libpng16 -DLODEPNG_NO_COMPILE_PNG -DLODEPNG_NO_COMPILE_DISK -Ilibrary/ -fPIC -Wall -Werror=implicit-function-declaration -pthread -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -o libpixbufloader-flif.so flif-pixbuf-loader.c -lpng16 -lgdk_pixbuf-2.0 -lgobject-2.0 -lglib-2.0 -L. -lflif
flif-pixbuf-loader.c:40:43: error: missing binary operator before token "("
 #if defined __USE_XOPEN2K8 || __GLIBC_USE (LIB_EXT2)
                                           ^
make: *** [Makefile:69: libpixbufloader-flif.so] Error 1
```

Running amd64 Gentoo

P.S. Guess I should ping @thebombzen

**Comment by thebombzen** [comment]
Apparently I had used the wrong feature test macro, but it compiled on my system anyway. Try replacing this line: 

    #if defined __USE_XOPEN2K8 || __GLIBC_USE (LIB_EXT2)

with this one (which is says is the right one in the manpages)

    #if (_XOPEN_SOURCE >= 700 || _POSIX_C_SOURCE >= 200809L)

If it works, I'll create a pull request to fix this.

**Comment by lehitoskin** [comment]
Yeah, no more compiler errors with that change.

**Comment by thebombzen** [comment]
Potential fix in #421

**Comment by thebombzen** [comment]
This issue was confirmed fixed in #421 and should be closed.
